reana/reana_dev/cluster.py

```diff
@@ -71,8 +75,15 @@ def cluster_commands():
     is_flag=True,
     help="Disable default CNI and use e.g. Calico.",
 )
+@click.option(
+    "--kind-version",
+    default=KIND_IMAGE_VERSION,
```

Comment:
Instead of providing a default, let's make this flag optional? If not provided, kind will decide on its own which version to use.

I'm saying this in particular because on kind releases page they even recommend to specify the hash of the images, as different images might not be compatible with all versions of kind ([source](https://github.com/kubernetes-sigs/kind/releases)):
_NOTE: You must use the sha256 digest to guarantee an image built for this release, until such a time as we switch to a different tagging scheme. Even then we will highly encourage digest pinning for security and reproducibility reasons._

So having a single default image might not work on every `kind` version.

What do you think?

Reply:
Agree, will make it optional.